src/components/BookPage/index.js

```diff
@@ -48,6 +48,8 @@ function BookPage() {
       })
       .catch((error) => console.error("Error fetching data", error));
   }, [id]);
+  function handleReviewSubmit(newReview) {
+    setReviews([...reviews, newReview])}
```

Comment:
I see why you're doing this, but a better way to do it would be to get handleReviewSubmit to just run the fetch request that's in your useEffect again. Just move that fetch request into it's own function that gets called in the useEffect, and gets called on review form submission instead of handleReviewSubmit.

Better for two reasons:

1) It's a bit simpler (Arguable)
2) It makes sure the data is always accurate

Reply:
done